rules.neon

```diff
@@ -22,6 +22,10 @@ parameters:
 		uselessCast: %strictRules.allRules%
 		requireParentConstructorCall: %strictRules.allRules%
 		disallowedConstructs: %strictRules.allRules%
+		disallowedBacktick: %strictRules.disallowedConstructs%
```

Comment:
These lines should say `%strictRules.allRules%` as it does on all other lines.

Reply:
Ah, this is going to be in 2.0, so we can break it.